### PR TITLE
EPMRPP-44255 || Test item history update

### PR DESCRIPTION
**Pull request by Yumfriez**

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java [hunk 1]
```diff
@@ -53,6 +54,18 @@
 	Page<TestItem> findByFilter(boolean isLatest, Queryable launchFilter, Queryable testItemFilter, Pageable launchPageable,
 			Pageable testItemPageable);
 
+	Page<TestItemHistory> loadItemsHistoryPage(Queryable filter, Pageable pageable, Long projectId, int historyDepth);
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java]: docs

File: src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java [hunk 2]
```diff
@@ -53,6 +54,18 @@
...
+
+	Page<TestItemHistory> loadItemsHistoryPage(Queryable filter, Pageable pageable, Long projectId, String launchName, int historyDepth);
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java]: docs

File: src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java [hunk 3]
```diff
@@ -53,6 +54,18 @@
...
+
+	Page<TestItemHistory> loadItemsHistoryPage(Queryable filter, Pageable pageable, Long projectId, List<Long> launchIds, int historyDepth);
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java]: docs

File: src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java [hunk 4]
```diff
@@ -53,6 +54,18 @@
...
+
+	Page<TestItemHistory> loadItemsHistoryPage(boolean isLatest, Queryable launchFilter, Queryable testItemFilter, Pageable launchPageable,
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java]: docs

File: src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java [hunk 5]
```diff
@@ -53,6 +54,18 @@
...
+			Pageable testItemPageable, Long projectId, int historyDepth);
+
+	Page<TestItemHistory> loadItemsHistoryPage(boolean isLatest, Queryable launchFilter, Queryable testItemFilter, Pageable launchPageable,
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java]: docs

File: src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java (hunk 1 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java]: added

File: src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java (hunk 2 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java]: added

File: src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java (hunk 3 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java]: added

File: src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java (hunk 5 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java]: added

File: src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java (hunk 4 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustom.java]: added

**Pull request merged**